src/core/public/core_system.ts

```diff
@@ -95,8 +99,9 @@ export class CoreSystem {
       this.rootDomElement.appendChild(this.notificationsTargetDomElement);
       this.rootDomElement.appendChild(this.legacyPlatformTargetDomElement);
 
-      const notifications = this.notifications.start();
       const injectedMetadata = this.injectedMetadata.start();
+      this.i18nService.start({ injectedMetadata });
```

Comment:
Initialization of i18n engine should be before Notifications start.

Reply:
Looks like we don't need `src/ui/public/chrome/api/translations.js` anymore, can you please remove it (including its usage in `src/ui/public/chrome/chrome.js`)?

Reply:
And I believe usage of `i18n.init` in `src/ui/ui_render/bootstrap/app_bootstrap.js` (and hence usage of `await server.getUiTranslations()` in `src/ui/ui_render/ui_render_mixin.js`) isn't not needed anymore as well. Can you please double check and remove everything redundant where possible? With this we'll have just 2 places where we init `i18n` - one for server and one for client.

Reply:
@azasypkin sure.
Done.